src/components/ui/viewPager/viewPager.component.tsx

```diff
@@ -191,14 +195,20 @@ export class ViewPager<ChildrenProps = {}> extends React.Component<ViewPagerProp
   };
 
   public render(): React.ReactElement<ViewProps> {
-    const { style, children, ...viewProps } = this.props;
+    const { style, children, swipeEnabled, ...viewProps } = this.props;
+
+    const animatedViewProps = swipeEnabled ? {
+      ...viewProps,
+      ...this.panResponder.panHandlers,
+    } : {
+      ...viewProps,
+    };
```

Comment:
Let's simplify a little
```suggestion
        const panResponderConfig = swipeEnabled ? this.panResponder.panHandlers : null;
        const animatedViewProps = { ... viewProps, ...panResponderConfig  };
```